### ISSUE TITLE
Provide more context for audit failures

### ISSUE THREAD

**Issue by alice** [GoogleChrome/accessibility-developer-tools#21]
For example, for color contrast failures, provide the same information that's provided in the Accessibility sidebar in the Accessibility Developer Tools extension (calculated contrast radio, bgColor, fgColor, suggested color pairs for each target contrast ratio value).

**Comment by alice** [comment]
Note: once this is addressed we no longer need both `NonExistentAriaRelatedElement` and `BadAriaAttributeValue`, so these should be merged into one rule.
